pages/pipelines/trigger_step.md.erb

```diff
@@ -206,3 +206,31 @@ To set environment variables on the build created by the trigger step, use the `
       RELEASE_STREAM: "${RELEASE_STREAM:-stable}"
 ```
 {: codeblock-file="pipeline.yml"}
+
+## Triggering specific steps in a pipeline
+
+While you cannot trigger only a specific step in a pipeline, you can use [conditionals](/docs/pipelines/conditionals) achieve a similar effect. A combination of the following environment variables might be useful:
```

Comment:
```suggestion
While you cannot trigger only a specific step in a pipeline, you can use [conditionals](/docs/pipelines/conditionals) to achieve a similar effect. A combination of the following environment variables might be useful:
```
In addition to conditionals you could also use [dynamic pipelines](https://buildkite.com/docs/pipelines/defining-steps#dynamic-pipelines), a little more complicated but might suit folks who are already using them.